batavia/core/exceptions.js

```diff
@@ -432,17 +477,17 @@ exceptions.ValueError = ValueError.prototype.__class__
 
 exceptions.Warning = undefined
 
-var ZeroDivisionError = function(msg) {
-    Exception.call(this, 'ZeroDivisionError', msg)
+var ZeroDivisionError = function() {
+    Exception.call(this, 'ZeroDivisionError', Array.from(arguments))
 }
 ZeroDivisionError.prototype = Object.create(Exception.prototype)
 ZeroDivisionError.prototype.__class__ = new Type('ZeroDivisionError', [Exception.prototype.__class__])
 ZeroDivisionError.prototype.__class__.$pyclass = ZeroDivisionError
 
 exceptions.ZeroDivisionError = ZeroDivisionError.prototype.__class__
 
-var JSONDecodeError = function(msg) {
-    Exception.call(this, 'JSONDecodeError', msg)
+var JSONDecodeError = function() {
```

Comment:
How do you feel about using the ES6 rest syntax here?

```
var JSONDecodeError = function(...args) {
    Exception.call(this, 'JSONDecodeError', args)
}
```

arguments feels magic and having to do Array.from is unfortunate.